src/components/Photo/SelfieIntro.js

```diff
@@ -41,7 +41,10 @@ class Intro extends Component<Props, State> {
       <div className={theme.thickWrapper}>
         <Button
           variants={['primary', 'centered', 'lg']}
-          onClick={nextStep}
+          onClick={() => {
+            sendScreen(['face_selfie_capture'])
```

Comment:
What's the full event name that woopra will receive? I just want to make sure we are not sending a duplicate event name

Reply:
The full name will be `screen_face_selfie_capture`. I added this event because I did not see it was already present for either woopra or user analytics, please let me know if that's incorrect.

Reply:
These are the events we are currently present in woopra:
![Screenshot 2020-07-09 at 18 15 41](https://user-images.githubusercontent.com/7127427/87070183-546e4700-c210-11ea-8555-74b3553221dc.png).
I think `screen_face_selfie_capture_camera` is sent when the user access the camera, while `screen_face_selfie_capture_file_upload` is sent when the user sees the uploader screen. I think it's fine to send the 'face_selfie_capture'. I will approve

Reply:
Thanks @stefaniacardenas, I wasn't able to find `screen_face_selfie_capture_camera` in my local testing. Please let me know if you want me to update.

Reply:
I think it's ok to just send `screen_face_selfie_capture`